source/hardware/EventDrivenGP.h

```diff
@@ -147,7 +147,14 @@ namespace emp {
 
       Event & operator=(const Event &) = default;
       Event & operator=(Event &&) = default;
-
+      bool operator<(const Event & other) const {
+        return std::tie(id, affinity)
+          < std::tie(other.id, other.affinity);
+      }
```

Comment:
I'd mention in the docstring that we're not comparing by one of the elements (the message?)

Reply:
done in 
b41cd2c